shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/domain/model/StudyPlanSectionType.kt

```diff
@@ -0,0 +1,21 @@
+package org.hyperskill.app.study_plan.domain.model
+
+enum class StudyPlanSectionType(val value: String) {
+    STAGE("stage"),
+    WRAP_UP_PROJECT("wrap up your project"),
+    WRAP_UP_TRACK("wrap up your track"),
+    EXTRA_TOPICS("extra topics"),
+    NEXT_PROJECT("next project"),
+    ROOT_TOPICS("root topics"),
+    NEXT_TRACK("next track");
+
+    companion object {
+        private val VALUES: Array<StudyPlanSectionType> = values()
+
+        fun getByValue(value: String): StudyPlanSectionType? =
+            VALUES.firstOrNull { it.value == value }
```

Comment:
Instead of writing this code we could make `StudyPlanSectionType` serializable. What are advantages of this approach?